src/include/daos/placement.h

```diff
@@ -80,6 +79,30 @@ struct pl_obj_layout {
 	struct pl_obj_shard	*ol_shards;
 };
 
+/** common header of all placement map */
+struct pl_map {
+	/** correpsonding pool uuid */
+	uuid_t			 pl_uuid;
+	/** link chain on hash */
+	d_list_t		 pl_link;
+	/** protect refcount */
+	pthread_spinlock_t	 pl_lock;
```

Comment:
The patch only moves the "struct pl_map" definition from src/placement/pl_map.h to src/include/daos/placement.h, but not change its inside members, then this patch will not affect the original "pl_map" related logic to avoid unnecessary side-effect.

If need to operation pl_ref via atomic_fetch_add/sub, we can do that in other separated patch later.